install.js

```diff
@@ -79,7 +79,11 @@ if (process.env.CLDR_URL) {
   );
 
 } else {
-
+  if (parentPackage["cldr-data-urlsjson"]) {
```

Comment:
Can you rename it to either `cldr-data-urls` only or `cldr-data-urls-json` please?